tspconfig.yaml

```diff
@@ -0,0 +1,16 @@
+emit:
+  - "@typespec/openapi3"
+  - "@azure-tools/typespec-ts"
+options:
+  "@typespec/openapi3":
+    "emitter-output-dir": "{project-root}/generated/openapi3"
+  "@azure-tools/typespec-ts":
+    generateMetadata: true
+    generateTest: false
+    azureSdkForJs: false
+    # isModularLibrary: true
```

Comment:
Currently I comment it out, because there's a bug in tcgc causing it failed to generate. 
<img width="782" alt="image" src="https://github.com/bterlson/openai-in-typespec/assets/55688292/d0105f2e-7d37-4623-9de8-524ee4966a94">